licenses/LICENSE_asm_bsd-3-clause

```diff
@@ -1,28 +0,0 @@
-
- ASM: a very small and fast Java bytecode manipulation framework
```

Comment:
this file had to be added explicitly as a copy of the licence had to be provided, and it's not added in the build-time version.

Reply:
We don't provide ASM, is is already included in the upstream agent. I think this was initially added because we use it as a compile-time dependency, for which don't need to include the licenses though because we don't ship them.